app/javascript/components/UserProvider.js

```diff
@@ -1,8 +1,8 @@
 import React, { useContext } from 'react'
 
 // window.SCP is not available when running via Jest tests,
-// so default such cases to a blank string
-const accessToken = 'SCP' in window ? window.SCP.userAccessToken : ''
+// so default such cases to a string "test"
+export const accessToken = 'SCP' in window ? window.SCP.userAccessToken : 'test'
```

Comment:
```suggestion
export const accessToken = ('SCP' in window) ? window.SCP.userAccessToken : 'test'
```
just because I never like having to remember order-of-operations for uncommon operators

Reply:
Also, if this is using `'SCP' in window` as a proxy for 'am I executing in a test environment', should it just be checking the node environment instead?

Reply:
Seems reasonable -- I've opened SCP-2237 for this and related tech debt.